apache/zeppelin
# [ZEPPELIN-1832] Fixed a bug in zombie process when Zeppelin stopped.

### What is this PR for?
When Zeppelin stops, there is a problem.
Sometimes this happens to the test as well.
It is related to releasing the resources of the remote remote interpreter, and its order has been modified.

### What type of PR is it?
Bug Fix

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1832
### How should this be tested?
1. Zeppelin start
2. run paragraph for interpreter (for example, %spark println("Hello world")
3. Zeppelin stop
4. check to zombie process 
```
#jps 
```
When Zeppelin exits, the RemoteInterpreter process should not exist.

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no
